hack/run-test.sh

```diff
@@ -12,6 +12,7 @@ usage() {
     print "    -h                Help for ${CURRENT_SCRIPT}"
     print "    -t                list of space separated paths to Testsuites to execute"
     print "    -p                string with extra Params for ginkgo"
+    print "    -r                string with report Params for ginkgo (these params will go after the list of suites)"
```

Comment:
proper positioning of ginkgo flags seems to be the responsability of this wrapper. So the wrapper can tolerate wrong order of flags, and the clarification between parens is unnecessary (a little layering violation)

Reply:
Two things here. 
While I have found problems when `--junit-report` is *before* `require-suite` I think @mrniranjan found some time ago, just the opposite,  that everything *after* `require-suite` was ignored by Ginkgo. I have been looking around but could not found anything backing any of the two ideas. 

Regarding, the wrapper being responsible of the param order, I am totally agree, in fact I do not like this approach but, as ginkgo params are passed as a string, for the wrapper to handle the proper order should have need to parse the params string and I thought that could lead to many problems, so this approach was simpler.